### PR TITLE
Sam

### PR DESCRIPTION
**Pull request by samliu21**
How does my code look @jeffreylin-21?

### CODE REVIEW

File: src/gr11review/part2/Utility.java [hunk 1]
```diff
@@ -1,5 +1,176 @@
...
+
+    // Declare variable
+    String strOutput;
+
+    // Initialize variable 
+    strOutput = "";
+
+    // Iterate through the string and check if the letter i is 'z' and the letter i+2 is 'p'. If so, add 'zp' to the output and increment i by 2 to skip to the letter after 'p'. If not, add character i to the output
+    for (int i = 0; i < strInput.length(); i++) {
+      if (i+2 < strInput.length() && strInput.charAt(i) == 'z' && strInput.charAt(i+2) == 'p') {
```
Comment (jeffreylin-21) [inline src/gr11review/part2/Utility.java]: Make sure you add spaces between operators, on line 23.

File: src/gr11review/part2/Utility.java [hunk 2]
```diff
@@ -1,5 +1,176 @@
...
+
+  }
+
+
+  // SAM
+  // Function that returns the longest word in a given text file
+  public static String longestWord(String strFileName) throws IOException {
+
+    // Create buffered reader 
+    BufferedReader thefile = new BufferedReader(new FileReader(strFileName));
```
Comment (jeffreylin-21) [inline src/gr11review/part2/Utility.java:53]: We might want to double-check with Mr. Fabroa where the file will be located.

File: src/gr11review/part2/Utility.java [hunk 3]
```diff
@@ -1,5 +1,176 @@
...
+
+  }
+
+
+  // SAM
+  // Function that prints the pascal triangle up to the given row i and column j
+  public static void pascalTri(int i, int j) throws IOException {
+
+    // Initialize print writer
+    PrintWriter theout = new PrintWriter(new FileWriter("src/gr11review/part2/pascalOut.txt"), false);
```
Comment (jeffreylin-21) [inline src/gr11review/part2/Utility.java:165]: Double-check if we should set this as false.

File: src/gr11review/part2/Utility.java [hunk 4]
```diff
@@ -1,5 +1,176 @@
...
+      for (intColCounter = 1; intColCounter < j; intColCounter++) {
+        intPascalTriangle[intRowCounter][intColCounter] = intPascalTriangle[intRowCounter - 1][intColCounter] + intPascalTriangle[intRowCounter][intColCounter - 1];
+      }
+    }
+
+    // Iterate through the 2D array and print out rows 0 to i-1 and columns 0 to j-1
+    for (intRowCounter = 0; intRowCounter < i; intRowCounter++) {
+      for (intColCounter = 0; intColCounter < j; intColCounter++) {
+        theout.print(intPascalTriangle[intRowCounter][intColCounter]);
+        if (intColCounter != j-1) {
```
Comment (jeffreylin-21) [inline src/gr11review/part2/Utility.java]: Add space between operators.

**Comment by samliu21** [comment]
Made your changes @jeffreylin-21

**Comment by samliu21** [comment]
Can you take another look? @jeffreylin-21

**Comment by jeffreylin-21** [comment]
> Can you take another look? @jeffreylin-21

Looks good to me.

**Pull request merged**